lib/common-types.ts

```diff
@@ -221,6 +221,19 @@ export interface MetadataProps {
   tags: Link<'Tag'>[]
 }
 
+export type MetadataAggeratedRootProps = {
```

Comment:
This is probably better to go into lib/entities/release.ts. You can then import here on line 82

Reply:
okie will do :)